x/erc20/keeper/ibc_callbacks.go

```diff
@@ -80,8 +80,12 @@ func (k Keeper) OnRecvPacket(
 		return ack
 	}
 
+	// Instead of converting just the received coins. We convert the whole user balance
+	balance := k.bankKeeper.GetBalance(ctx, recipient, coin.Denom)
```

Comment:
we need to check that it's > 0

Reply:
Didnt add the check beacause the balance already contains the coins received in the transfer

Reply:
> balance already contains the coins received in the transfer

Can you add that as a comment?